src/util/network.js

```diff
@@ -0,0 +1,52 @@
+import { KNOWN_CHAINS } from 'use-wallet'
+import { getNetworkConfig } from '../network-config'
+
+export function isOnEthMainnet(networkType) {
+  return networkType === KNOWN_CHAINS.get(1).type
+}
+
+export function isMainnet(networkType) {
+  return (
+    networkType === KNOWN_CHAINS.get(1).type ||
+    networkType === KNOWN_CHAINS.get(137).type
+  )
```

Comment:
I would like a property on network-config to indicate whether a network is testnet or mainnet instead of hardcoding the test here...